FunctionalTableData/CollectionView/FunctionalCollectionData.swift

```diff
@@ -202,7 +202,11 @@ public class FunctionalCollectionData: NSObject {
 					if $0.name == NSExceptionName.internalInconsistencyException {
 						guard let exceptionHandler = FunctionalTableData.exceptionHandler else { return }
 						let changes = TableSectionChangeSet()
-						let exception = FunctionalTableData.Exception(name: $0.name.rawValue, newSections: newSections, oldSections: strongSelf.sections, changes: changes, visible: [], viewFrame: strongSelf.collectionView?.frame ?? .zero, reason: $0.reason, userInfo: $0.userInfo)
+						var viewFrame: CGRect = .zero
```

Comment:
I think you can write this as:
`let viewFrame = DispatchQueue.main.sync { return strongSelf.collectionView?.frame ?? .zero }`